### PR TITLE
Removes AssertPermissionTrait

### PR DESCRIPTION
**Pull request by tankerkiller125**
Removes AssertPermissionTrait

### CODE REVIEW

File: src/Api/Controller/OrderTagsController.php [hunk 1]
```diff
@@ -10,22 +10,19 @@
...
 {
-    use AssertPermissionTrait;
-
     /**
      * {@inheritdoc}
      */
     public function handle(ServerRequestInterface $request): ResponseInterface
     {
-        $this->assertAdmin($request->getAttribute('actor'));
+        $request->getAttribute('actor')->isAdmin();
```
Comment (askvortsov1) [inline src/Api/Controller/OrderTagsController.php:25]: This should use ->assertAdmin

File: src/Command/CreateTagHandler.php [hunk 2]
```diff
@@ -40,7 +37,7 @@ public function handle(CreateTag $command)
         $actor = $command->actor;
         $data = $command->data;
 
-        $this->assertCan($actor, 'createTag');
+        $actor->can('createTag');
```
Comment (askvortsov1) [inline src/Command/CreateTagHandler.php:40]: This should be $actor->assertCan

File: src/Command/DeleteTagHandler.php [hunk 3]
```diff
@@ -41,7 +38,7 @@ public function handle(DeleteTag $command)
 
         $tag = $this->tags->findOrFail($command->tagId, $actor);
 
-        $this->assertCan($actor, 'delete', $tag);
+        $actor->can('delete', $tag);
```
Comment (askvortsov1) [inline src/Command/DeleteTagHandler.php:41]: This should be $actor->assertCan

File: src/Command/EditTagHandler.php [hunk 4]
```diff
@@ -50,7 +47,7 @@ public function handle(EditTag $command)
 
         $tag = $this->tags->findOrFail($command->tagId, $actor);
 
-        $this->assertCan($actor, 'edit', $tag);
+        $actor->can('edit', $tag);
```
Comment (askvortsov1) [inline src/Command/EditTagHandler.php:50]: This should be $actor->assertCan

**Review by askvortsov1 — CHANGES_REQUESTED** [review]

**Review by askvortsov1 — APPROVED** [review]

**Review by dsevillamartin — APPROVED** [review]

**Pull request merged**